apps/tokenomics/components/GlobalStyles/index.jsx

```diff
@@ -64,6 +69,9 @@ const GlobalStyle = createGlobalStyle`
   .ml-16 {
     margin-left: 16px;
   }
+  .mt-8 {
```

Comment:
have you thought on maybe doing some shared global styles in the mono repo root?